### ISSUE TITLE
Migrate to ghc-lib

### ISSUE THREAD

**Issue by fendor** [dan-t/hsimport#24]
Since haskell-src-exts is no longer actively maintained (https://github.com/haskell-suite/haskell-src-exts#maintenance), it will struggle to keep up with ghc releases. 
So, either this package stays with haskell-src-exts and will likely not work with newer ghc version or we migrate to ghc-lib. 

Would you be interested in a PR migrating to ghc-lib?

**Comment by dan-t** [comment]
On Tue, Mar 03, 2020 at 01:19:29AM -0800, fendor wrote:
> Would you be interested in a PR migrating to ghc-lib?

Yes.

Are you interested in taking over the maintainership of 'hsimport'?

**Comment by fendor** [comment]
Yes, I would be interested!

**Comment by dan-t** [comment]
On Tue, Mar 03, 2020 at 02:03:44AM -0800, fendor wrote:
> Yes, I would be interested!

Great! 
Should I transfer the 'hsimport' project to your 'fendor' github account?
I would also add you as a maintainer to 'hackage' and afterwards remove myself.
Is your 'hackage' name also 'fendor'?

**Comment by fendor** [comment]
Yes, that would be great! Yeah, I am fendor on hackage https://hackage.haskell.org/user/fendor

**Comment by dan-t** [comment]
On Tue, Mar 03, 2020 at 03:44:23AM -0800, fendor wrote:
> Yes, that would be great! Yeah, I am fendor on hackage https://hackage.haskell.org/user/fendor

It seems that I can't transfer it because of your 'hsimport' fork. Can you please remove the fork.

I would also transfer the 'hsimport-config' project to you, which is
only a minimal example for an user defined configuration.

I've added you as a maintainer to hackage.

**Comment by fendor** [comment]
> It seems that I can't transfer it because of your 'hsimport' fork. Can you please remove the fork.

Done! 

> I would also transfer the 'hsimport-config' project to you, which is
only a minimal example for an user defined configuration.

Makes sense, thank you! 

> I've added you as a maintainer to hackage.

Thank you!
